python/paddle/fluid/contrib/slim/quantization/imperative/quant_nn.py

```diff
@@ -209,13 +211,107 @@ def forward(self, input):
         return quant_out
 
 
-def _get_fake_quant_type(quant_type, name, moving_rate, quant_bits, dtype,
-                         quant_on_weight):
+class FakeChannelWiseQuantDequantAbsMax(layers.Layer):
+    def __init__(self,
+                 name=None,
+                 channel_num=None,
+                 quant_bits=8,
+                 quant_axis=0,
+                 dtype='float32',
+                 quant_on_weight=False):
```

Comment:
assert `quant_on_weight ` == False ?

Reply:
I think it should be assert quant_on_weight == True.

Reply:
Yes, you are right.